components/organisms/InsightPage/InsightPage.tsx

```diff
@@ -460,6 +460,7 @@ const InsightPage = ({ edit, insight, pageRepos }: InsightPageProps) => {
             isLoading={createLoading}
             placeholder="Repository URL or Full Name (ex: open-sauced/open-sauced)"
             className="!w-full text-md text-gra"
+            style={{ width: "100%" }}
```

Comment:
Refactor (blocking): The style is not required. The `w-full` Tailwind class does the same thing. It's already been used with a higher CSS specificity via `!w-full`.

It appears to not be applied for some reason, so I'd investigate that.

Reply:
> Refactor (blocking): The style is not required. The `w-full` Tailwind class does the same thing. It's already been used with a higher CSS specificity via `!w-full`.
> 
> It appears to not be applied for some reason, so I'd investigate that.

Yes, the !w-full isn’t working. So, I came up with the style prop.

Reply:
@RitaDee, what I meant is the `style` fix is masking the real issue, which is `!w-full` is not being applied, so I'd dig into that.

Reply:
@RitaDee the `!w-full` is applied to the div containing the input. The input itself has a width of `w-max` that's why it's not expanding to the full width. You change the `w-max` to `w-full` to fix the issue. You may also want to check all the places that component is used to make sure any other issue is caused

Reply:
> The input itself has a width of `w-max` that's why it's not expanding to the full width. You change the `w-max` to `w-full` to fix the issue. You may also want to check all the places that component is used to make sure any other issue is caused

Good call here @takanome-dev. We actually caught this in the discord call this morning with @nickytonline

Reply:
> Refactor (blocking): The style is not required. The `w-full` Tailwind class does the same thing. It's already been used with a higher CSS specificity via `!w-full`.
> 
> It appears to not be applied for some reason, so I'd investigate that.

@nickytonline, thank you for the walkthrough earlier. Having understood that the project should not go beyond the tailwind class, I opted to replace w-max with w-full in the Search component, and I have checked other places where the Search component is used or called, but I can’t find any regression or weird display. Can you check this out in my current commit?

Thank you @takanome-dev.